### ISSUE TITLE
[Bug] Cannot save a newly row (inline editor) 🚨

### ISSUE THREAD

**Issue by n-sviridenko** [nocodb/nocodb#2193]
**Please enter the following details**

```
Node: **v12.22.12**
Arch: **x64**
Platform: **linux**
Docker: **false**
Database: **pg**
ProjectOnRootDB: **false**
RootDB: **pg**
PackageVersion: **0.90.11**
```

**Steps To Reproduce**

https://www.loom.com/share/2518662ec019410e85684ca95ced2090

**Expected Behavior**

The newly created row is saved.

**Screenshots**

See the loom above.

<img width="641" alt="Screenshot 2022-05-28 at 16 16 14" src="https://user-images.githubusercontent.com/9335422/170829285-bbe29472-eeab-4f5e-87ad-b8c007e12f71.png">

**Comment by wingkwong** [comment]
Can you take a screenshot of the column settings of `Duration_seconds` just like below?

![image](https://user-images.githubusercontent.com/35857179/171097764-b6e79fbb-1473-4243-ba71-c0a816674d9f.png)

**Comment by wingkwong** [comment]
This has been fixed in `v0.98.1`. A new row will be only saved when all required fields are entered.

![image](https://user-images.githubusercontent.com/35857179/197944363-37304b06-8975-42fd-bc72-9468bdc802b7.png)
